examples/fluids/navierstokes.c

```diff
@@ -129,17 +150,41 @@ testData testOptions[] = {
     .testtol = 0.,
     .filepath = NULL
   },
-  [TEST_EXPLICIT] = {
+  [TEST_DC_EXPLICIT] = {
     .testtol = 1E-5,
-    .filepath = "examples/fluids/tests-output/fluids-navierstokes-explicit.bin"
+    .filepath = "examples/fluids/tests-output/fluids-navierstokes-dc-explicit.bin"
   },
-  [TEST_IMPLICIT_STAB_NONE] = {
+  [TEST_DC_IMPLICIT_STAB_NONE] = {
     .testtol = 5E-4,
-    .filepath = "examples/fluids/tests-output/fluids-navierstokes-implicit-stab-none.bin"
+    .filepath = "examples/fluids/tests-output/fluids-navierstokes-dc-implicit-stab-none.bin"
   },
-  [TEST_IMPLICIT_STAB_SUPG] = {
+  [TEST_DC_IMPLICIT_STAB_SUPG] = {
     .testtol = 5E-4,
-    .filepath = "examples/fluids/tests-output/fluids-navierstokes-implicit-stab-supg.bin"
+    .filepath = "examples/fluids/tests-output/fluids-navierstokes-dc-implicit-stab-supg.bin"
```

Comment:
What do you think of making the tolerance and output filepath a command-line option and then you wouldn't need this enum and initialization that only sets these two values? (I see the value if you have "cases" that set initial and boundary conditions, but here it looks like the command-line is already needed for everything else.)

Reply:
Yeah, I figured it was getting really messy but then I thought maybe we could create an enum for each problem after we are done with the refactoring! Even in that case though, the more capabilities we add to the mini-app, the more tests would be needed and it would get messy anyway!

Reply:
How about drop the enums and pass `-compare_final_state_atol 1e-11 -compare_final_state_filename tests-output/fluids-navierstokes-dc-implicit-stab-supg.bin`? Then the specification would be self-contained in the runtime options.